frontend/tests/components/LoginModal.test.tsx

```diff
@@ -104,28 +84,12 @@ describe("LoginModal", () => {
       />,
     );
 
-    const loginButton = screen.getByText("Sign In");
-    fireEvent.click(loginButton);
-
-    expect(setItemSpy).not.toHaveBeenCalled();
-  });
+    const signInButton = screen.getByText("Sign In");
+    fireEvent.click(signInButton);
 
-  it("should render the login button with correct text", () => {
```

Comment:
curious why you got rid of this test and the "should not store URL in session storage if pathname and search are empty" test. These both seem potentially useful to me if there's not a technical reason to take them out